lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -56,44 +55,187 @@ const ProgressEvent = require("../generated/ProgressEvent");
 const TouchEvent = require("../generated/TouchEvent");
 const UIEvent = require("../generated/UIEvent");
 
+const { ResourceLoader } = require("../../browser/new-resource-loader");
+const { wrapCookieJarForRequest } = require("../xhr-utils");
```

Comment:
Seeing how wrapCookieJarForRequest is used, I think it does not belong in xhr-utils. Maybe its own file inside living/helpers/?